arangod/IResearch/IResearchAnalyzerFeature.cpp

```diff
@@ -57,6 +58,7 @@
 namespace {
 
 static std::string const ANALYZER_COLLECTION_NAME("_iresearch_analyzers");
+static char const ANALYZER_PREFIX_DELIM = ':'; // name prefix delimiter (2 chars)
```

Comment:
the comment says `2 chars`, put the delimiter is just one char long...

Reply:
true, it's simpler to compare 1 char at a time